src/mainviews/job-definition.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useMemo } from 'react';
```

Comment:
If this file is intended for use only in the detail view, perhaps it should be at `mainviews/detail-view/job-definition.tsx`

Reply:
I agree. @andrii-i Let's refactor this to the following component hierarchy:

```
mainviews/detail-view
  |_ index.tsx (just exports from ./detail-view.tsx)
  |_ detail-view.tsx
  |_ job-detail.tsx
  |_ job-definition-detail.tsx
```